Can't generate SSH keys

After enabling Projects, I'm having issues when trying to generate the SSH keys.

To generate the keys, you must do the following.

Settings > Git config > SSH Keys > add key, fill up the Add SSH Key form and click on Generate key.

Node-RED won't show anything and the add-on will restart automatically.

Logs:

```
18 Dec 10:33:55 - [red] Uncaught Exception:
18 Dec 10:33:55 - Error: spawn ssh-keygen ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:362:16)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hassio-addon-node-red@ start: `node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hassio-addon-node-red@ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-12-18T15_33_55_550Z-debug.log

> hassio-addon-node-red@ start /opt
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"

18 Dec 10:33:56 - [info] 

Welcome to Node-RED
===================

18 Dec 10:33:56 - [info] Node-RED version: v0.19.5
18 Dec 10:33:56 - [info] Node.js  version: v8.14.0
18 Dec 10:33:56 - [info] Linux 4.9.0-8-amd64 x64 LE
18 Dec 10:33:56 - [info] Loading palette nodes
18 Dec 10:33:56 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
18 Dec 10:33:57 - [info] Settings file  : /etc/node-red/config.js
18 Dec 10:33:57 - [info] Context store  : 'default' [module=memory]
18 Dec 10:33:57 - [info] User directory : /config/node-red/
18 Dec 10:33:57 - [info] Server now running at http://127.0.0.1:1880/
18 Dec 10:33:57 - [info] Active project : my-node-red-config
18 Dec 10:33:57 - [info] Flows file     : /config/node-red/projects/my-node-red-config/flows.json
18 Dec 10:33:57 - [info] Starting flows
78:E1:03:B8:B5:C8
78:E1:03:1A:7C:50
18 Dec 10:33:59 - [info] Started flows
18 Dec 10:33:59 - [info] [server:Home Assistant] WebSocket Connected to http://hassio/homeassistant
```

--- a/node-red/Dockerfile
+++ b/node-red/Dockerfile
@@ -26,7 +26,7 @@ RUN \
         git=2.18.1-r0 \
         nodejs=8.14.0-r0 \
         npm=8.14.0-r0 \
-        openssh-keygen=7.7_p1-r3 \
+        openssh-client=7.7_p1-r3 \
         paxctl=0.9-r0 \
         python2=2.7.15-r1 \
     \

--- a/node-red/Dockerfile
+++ b/node-red/Dockerfile
@@ -26,6 +26,7 @@ RUN \
         git=2.18.1-r0 \
         nodejs=8.14.0-r0 \
         npm=8.14.0-r0 \
+        openssh-keygen=7.7_p1-r3 \
         paxctl=0.9-r0 \
         python2=2.7.15-r1 \
     \

